src/components/MatchScore/matchScore.module.scss

```diff
@@ -0,0 +1,33 @@
+$indicator-size: 12px;
+
+.matchScoreContainer {
```

Comment:
Use kebab case here match-score-container. I know the guidlines say to use camelCase, will update the documentation. You can still use camelCAse in ts